lib/debugger/debugger.dart

```diff
@@ -170,11 +168,21 @@ class DebuggerScreen extends Screen {
 
     _sourcePathDiv.setInnerHtml('&nbsp;');
 
+    // configure the navigation / editor splitter
     split.flexSplit(
-      panels,
-      gutterSize: 12,
-      sizes: [25, 75],
-      minSize: [150, 200],
+      navEditorPanels,
+      gutterSize: defaultSplitterWidth,
+      sizes: [22, 78],
+      minSize: [200, 600],
```

Comment:
strangely, the chrome-devtools allow very small min-sizes for the text editor and stack frame windows. However, I agree that these more practical minSizes are worth a try at least until the UI actually looks alright with smaller widths.

Reply:
Yeah, happy to tweak these going forward, esp. if we find the min sizes cause issues for users.